static/js/components/search/CustomSortingColumnHeaders.js

```diff
@@ -1,15 +1,19 @@
 // @flow
+/* global SETTINGS: false */
 import React from 'react';
+import { connect } from 'react-redux';
 import Grid, { Cell } from 'react-mdl/lib/Grid';
 import R from 'ramda';
 
+import { canAdvanceSearchProgram } from '../../lib/roles';
 import type { SearchSortItem } from '../../flow/searchTypes';
+import type { AvailableProgram } from '../../flow/enrollmentTypes';
 
 const nameKeys = ['name_a_z', 'name_z_a'];
 const locationKeys = ['loc-a-z', 'loc-z-a'];
 const gradeKeys = ['grade-high-low', 'grade-low-high'];
 
-export default class CustomSortingColumnHeaders extends React.Component {
+export class CustomSortingColumnHeaders extends React.Component {
```

Comment:
Are we using the exported version of this class here? If not we should remove it so we don't confuse it for the default export